Source/SwiftLintFramework/Rules/EmptyParenthesesWithTrailingClosureRule.swift

```diff
@@ -24,7 +24,12 @@ public struct EmptyParenthesesWithTrailingClosureRule: ASTRule, ConfigurationPro
             "[1, 2].map({ $0 + 1 })\n",
             "[1, 2].reduce(0) { $0 + $1 }",
             "[1, 2].map { number in\n number + 1 \n}\n",
-            "let isEmpty = [1, 2].isEmpty()\n"
+            "let isEmpty = [1, 2].isEmpty()\n",
+            "UIView.animateWithDuration(0.3, animations: {\n" +
+            "   self.disableInteractionRightView.alpha = 0\n" +
+            "}, completion: { _ in\n" +
+            "   ()\n" +
```

Comment:
thanks for fixing this "useless" case, it is valid Swift :P
we tend to use () in our switch statement for no-op instead of "break" since break can mean breaking out of a loop